lxc-binding.c

```diff
@@ -52,6 +53,9 @@ bool go_lxc_want_close_all_fds(struct lxc_container *c, bool state) {
 }
 
 bool go_lxc_create(struct lxc_container *c, const char *t, const char *bdevtype, int flags, char * const argv[]) {
+	if (strncmp(t, "none", strlen(t)) == 0) {
+		return c->create(c,NULL,bdevtype, NULL, !!(flags & LXC_CREATE_QUIET), argv);
```

Comment:
This doesn't look like formatting that gofmt would agree with, I'd expect spaces after those commas.

Reply:
Indeed. My bad!